packages/components/Button/src/Button.tsx

```diff
@@ -25,6 +25,15 @@ export const Button = compose<IButtonType>({
     } = userProps;
     // attach the pressable state handlers
     const pressable = useAsPressable({ ...rest, onPress: onClick });
+    const onKeyDown = React.useCallback(
+      e => {
+        if (onClick && e.nativeEvent.key === 'Enter') {
```

Comment:
>Enter [](start = 46, length = 5)

What about space key???

Reply:
Added!

Reply:
Actually, usually buttons only support space, not enter.  We should check to make sure this is the expected design.

Reply:
Or maybe that's only true for other widgets like Checkboxes?  Disregard :)

Reply:
I checked buttons on Office apps, and they supported enter!